app/src/main/java/com/example/criminalintentapp/presentation/fragments/crime_list/CrimeListViewModel.kt

```diff
@@ -3,8 +3,7 @@ package com.example.criminalintentapp.presentation.fragments.crime_list
 import androidx.lifecycle.ViewModel
 import com.example.criminalintentapp.data.repository.CrimeRepository
 
-class CrimeListViewModel : ViewModel() {
+class CrimeListViewModel(private val crimeRepository: CrimeRepository) : ViewModel() {
```

Comment:
"Constructor parameter is never used as a property" -> you can remove `private val`